eeps/eep-0054.md

```diff
@@ -1,5 +1,5 @@
     Author: Björn Gustavsson <bjorn(at)erlang(dot)org>
-    Status: Final/OTP-24 Implemented in OTP 24
+    Status: Final/24.0 Implemented in OTP release 24.0
     Type: Standards Track
     Created: 14-Sep-2020
     Erlang-Version: 24
```

Comment:
I'm not able to suggest the Post-History here.
```
    Post-History: 14-Oct-2020, 27-Nov-2020, 06-Sep-2023
```

Reply:
Probably no specific preceding public discussion